pages/treeherder.py

```diff
@@ -17,6 +17,7 @@ class TreeherderPage(Base):
 
     _result_set_locator = (By.CSS_SELECTOR, '#th-global-content .result-set')
     _selected_job_title_locator = (By.CSS_SELECTOR, '.job-list .selected-job')
+    _pin_all_jobs_locator = (By.CSS_SELECTOR, '.th-view-content > ng-view:nth-child(1) > div:nth-child(2) > div:nth-child(1) > span:nth-child(3) > span:nth-child(2) > span:nth-child(1)')
```

Comment:
This looks like a fragile locator. I think `.result-set .glyphicon-pushpin` would work just as well. I actually think it would be worth creating a `ResultSet` page region, which would allow us to target any result set's children elements.